### DISCUSSION TITLE
Cannor run httpx command

### DISCUSSION THREAD

**Discussion by nadizeq** [projectdiscovery/httpx#582] [Q&A] [answered]
After installing the installation instruction as per to github which I list it below.
`go install -v github.com/projectdiscovery/httpx/cmd/httpx@latest
`
The command is a success but when I try to run httpx -h, it shows this

![image](https://user-images.githubusercontent.com/80656694/163132488-51eee8c6-7d29-4dab-93fd-29b34b9cc7e9.png)

**Comment by nadizeq** [comment]
GO111MODULE=on go install -v https://github.com/projectdiscovery/httpx.git

Output:
malformed import path "https:/github.com/projectdiscovery/httpx.git": invalid char ':'

**Comment by DonatoReis** [comment]
@nadizeq  did you move the binary to /usr/local/bin ???

**Comment by ehsandeep** [comment, marked as answer]
@nadizeq you can install with:

```
git clone https://github.com/projectdiscovery/httpx.git; cd httpx/cmd/httpx; go build; cp httpx /usr/local/bin; httpx -version
```
